python/graphstorm/model/utils.py

```diff
@@ -738,7 +744,86 @@ def save_shuffled_node_embeddings(shuffled_embs, save_embed_path, save_embed_for
 
     if rank == 0:
         with open(os.path.join(save_embed_path, "emb_info.json"), 'w', encoding='utf-8') as f:
-            f.write(json.dumps(emb_info))
+            json.dump(emb_info, f, indent=4)
+
+def save_full_node_embeddings(g, save_embed_path,
+                              embeddings,
+                              node_id_mapping_file,
+                              save_embed_format="pytorch"):
+    """ Save full node embeddings through pytorch in a distributed way.
+
+        `save_full_node_embeddings` will save two information of an embedding: 1)
+        the embedding and 2) its corresponding node ID in GraphStorm node ID space.
```

Comment:
what is the GraphStorm node ID space? i thought the node IDs are very original node IDs from users' input data.